runtime/oti/j9nonbuilder.h

```diff
@@ -5119,6 +5119,13 @@ typedef struct J9JavaVM {
 	 * of the feature constants listed in j9port.h
 	 */
 	U_32 cpuCacheWritebackCapabilities;
+	/* Used to parse -XX:+UseLargePages, and -XX:LargePageSizeInBytes=<size> 
+	 * so that both the GC(Objectheap) and JIT(Codecache) are able to access these attributes.
+	 * LargePageSizeRequested - <size> element parsed in the case of -XX:LargePageSizeInBytes.
+	 * largePageArgIndex - The right most index of either option.
+	 */
+	IDATA largePageSizeRequested;
+	IDATA largePageArgIndex;
```

Comment:
I am not sure that J9VM is a best place to pass parameters but I am ok if VM developers can tolerate this

Reply:
@pshipton Could you confirm this?

Reply:
I'm not a huge fan of this approach but can live with it if the two fields are wrapped in an inline struct.  

Please introduce a new struct something like the following to hold the two values:
```c
typedef struct J9LargePageCompatibiltyOptions {
    IDATA pageSizeRequested;
    IDATA optionIndex;
}
```